.human/TASK_LIST.md

```
@@ -0,0 +1,23 @@
+# Human Action Items & Strategy (Marcelo / Zen Eyer)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)